docs/design/managing-resources.md

```diff
@@ -8,12 +8,14 @@ Cross-object field references are a way for an object to get output field(s) of
 (or multiple dependencies) injected into it as a field (or multiple fields). References to only direct
 dependencies are supported.
 
-Syntax `"$({dependency1/fieldName})"` means that value of `fieldName` will be injected as a string instead
+Syntax `"$(dependency1#fieldName)"` means that value of `fieldName` will be injected as a string instead
 of the placeholder. In this case value must be a string, boolean or number.
 
-Syntax `"$(({dependency1/fieldName}))"` means that value of `fieldName` will be injected without quotes
+Syntax `"$((dependency1#fieldName))"` means that value of `fieldName` will be injected without quotes
 instead of the placeholder. In this case it can be a of any type including objects.
 
+The `fieldName` could be specified in JsonPath format (with `$.` prefix added by default), for example: `$(dependency1#status.conditions[?(@.type=="Ready")].status)`
```

Comment:
See the example of complex JsonPath reference:
`$(dependency1#status.conditions[?(@.type=="Ready")].status)`